packages/mds-policy/api.ts

```diff
@@ -361,6 +377,31 @@ function api(app: express.Express): express.Express {
       })
   })
 
+  app.post(pathsFor('/admin/policies/:policy_id/publish'), (req, res) => {
```

Comment:
I may be misunderstanding the behavior of this endpoint, but it looks like you're writing an unpublished policy to the db, not publishing it. Needs a rename

Reply:
You are so right, I derped there.